metaflow/plugins/kfp/kfp.py

```diff
@@ -108,8 +108,15 @@ def create_kfp_pipeline_yaml(
         Creates a new KFP pipeline YAML using `kfp.compiler.Compiler()`.
         Note: Intermediate pipeline YAML is saved at `pipeline_file_path`
         """
+        pipeline_conf = PipelineConf()
+        pipeline_conf.set_timeout(self.workflow_timeout)
+        if ARGO_DEFAULT_TTL is not None:  # if None, we use the Argo defaults
```

Comment:
@talebzeghmi looking back on it, `ARGO_DEFAULT_TTL` is a confusing name, because it actually refers to the TTL default defined in the Metaflow config file, but seems to refer to the value set in the Argo manifests. I'll make a PR to change this to `FROM_CONF_TTL`?

Reply:
I believe `FROM_CONF_TTL` is even more confusing.
Why not: `KFP_TTL_SECONDS_AFTER_FINISHED`

Reply:
True `ARGO_DEFAULT_TTL` isn't descriptive enough.